src/WebJobs.Script/Extensions/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Extensions
+{
+    internal static class ConfigurationExtensions
+    {
+        internal static IConfigurationSection GetWebJobsConnectionStringSection(this IConfiguration configuration, string connectionStringName)
```

Comment:
Where are these new methods used in this PR - I don't see them being called (did a string search in PR)? This one and the other GetConnectionStringOrSettingSection below.

Reply:
Good point! Removed! Somewhat included by mistake!